reconcile/external_resources/model.py

```diff
@@ -220,13 +223,46 @@ def load_module_inventory(
     })
 
 
+class ResourcesSpec(BaseModel, frozen=True):
+    cpu: str = ""
+    memory: str = ""
+
+
+class Resources(BaseModel, frozen=True):
+    """Hashable class to store module resources in reconciliations.
+    Default values are used as a fallback for existent objects that were
+    created without container resources, hence they don't have mem/cpu resources
+    in the ERv2 State. Eventually, all resources will have resources assignments
+    from the module spec, module_overrides, or app-interface settings.
+    """
+
+    requests: ResourcesSpec = ResourcesSpec(cpu="100m", memory="128Mi")
+    limits: ResourcesSpec = ResourcesSpec(memory="4Gi")
+
+    @staticmethod
+    def from_deploy_resources_fields(fields: DeployResourcesFields) -> "Resources":
+        """Create Resource obect from GQL DeployResourcesFields.
+
+        DeployResourceFields can not be used directly as it not hashable."""
+        return Resources(
+            requests=ResourcesSpec(
+                cpu=fields.requests.cpu or "", memory=fields.requests.memory or ""
+            ),
+            limits=ResourcesSpec(
+                cpu=fields.limits.cpu or "",
+                memory=fields.limits.memory or "",
+            ),
+        )
```

Comment:
I don't understand the default settings for `ResourcesSpec.cpu`, `ResourcesSpec.memory`, `Resources.requests`, `Resources.limits` and everything in `from_deploy_resources_fields`. Are the defaults `""` or 

```
requests: ResourcesSpec = ResourcesSpec(cpu="100m", memory="128Mi")
limits: ResourcesSpec = ResourcesSpec(memory="4Gi")
```

Reply:
I changed this code. PTAL.

Reply:
Now, it's much easier to understand. Thank you 🙇